docs/en/writing-running-appium/other/how_to_get_appium_server_logs.md

```diff
@@ -0,0 +1,18 @@
+# How to get Appium server logs
+  If there's an issue with a test execution, the Appium server logs are normally the best way to find the root of the issue. This document shows how to get the Appium server logs and a short explaination of how to read them.
+
+## Using the [desktop GUI](https://github.com/appium/appium-desktop)
+  There is a button in the desktop GUI which will export the logs to a file. 
```

Comment:
Instead of saying "There is a button" you can mention it as "Get Raw Logs" button